simpleui-server/src/props-file-reader.ts

```diff
@@ -353,6 +350,7 @@ export class PropsFileReader {
                         props[key][index]['index'] = (keyIndex - 1).toString();
                         props[key][index]['id'] = `${key}-${index + 1}`;
                         props[key][index][subKey] = PropsFileReader.replaceMacros(props, value);
+                        let wwwwww = props[key][index][subKey]
```

Comment:
I don't think wwwwww is used anywhere else. Was this for debugging?

Reply:
oops, yea that was for debugging. I removed it in my last commit